plugins/packages/re-indexer/main.go

```diff
@@ -0,0 +1,138 @@
+package main
+
+import (
+	// "fmt"
+	"context"
+	"database/sql"
+	"os"
+	"strings"
+	"net/http"
+	"time"
+	"encoding/json"
+	// "reflect"
+
+	"github.com/gorilla/websocket"
+	log "github.com/inconshreveable/log15"
+	"github.com/openrelayxyz/cardinal-streams/transports"
+	// "github.com/openrelayxyz/cardinal-streams/delivery"
+	"github.com/openrelayxyz/cardinal-types/hexutil"
+	// "github.com/openrelayxyz/cardinal-evm/rlp"
+	"github.com/openrelayxyz/flume/indexer"
+	"github.com/openrelayxyz/flume/config"
+	"github.com/openrelayxyz/flume/plugins"
+)
+
+func Initialize(cfg *config.Config, pl *plugins.PluginLoader) {
+	log.Info("Re-indexer loaded")
+}
+
+type message struct {
+	Id int          `json:"id"`
+	Method string   `json:"method"`
+	Params []string `json:"params"`
+}
+
+type resultMessage struct {
+	Type string `json:"type"`
+	Batch *transports.TransportBatch `json:"batch,omitempty"`
+}
+
+type outerResult struct {
+	Result  *resultMessage `json:"result"`
+	JsonRPC string         `json:"jsonrpc"`
+	Id		int			   `json:"id"`
+}
+
+func ReIndexer(cfg *config.Config, db *sql.DB, indexers []indexer.Indexer) error {
+
+	var wsURL string
+
+	for _, broker := range cfg.BrokerParams {
+		if strings.HasPrefix(broker.URL, "ws://") || strings.HasPrefix(broker.URL, "wss://") {
+			wsURL = broker.URL
+			log.Info("found websocket broker, reindexer", "broker", wsURL) 
```

Comment:
Probably go ahead and stick a break statement in here so we don't look at the rest of the brokers.

Reply:
Break after the log, or the log won't happen